src/app/users/users-table.component.html

```diff
@@ -130,6 +130,9 @@
     <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
     <mat-row [ngClass]="{'cursor-pointer':!isDialog}" *matRowDef= "let row; columns: displayedColumns;" (click)="gotoProfileView(row.doc.name)" [ngClass]="{highlight:isSelected(row.doc)}"></mat-row>
   </mat-table>
+  <ng-template #noUserFoundMessage>
+    <div class="view-container">No members available</div>
```

Comment:
`view-container` should not be used here.  It already applies to the containing element.